src/scripts.js

```diff
@@ -78,33 +85,89 @@ const parseData = (data) => {
   //below functions will move to domManip on click events
   session.getTotalDollarSpentThisYear()
   session.getTripCost('Cartagena, Colombia', 7, 2)
+  addDestinationSearch()
   // session.getPastTrips(todayDate)
-  changeFormView()
+  // changeFormView(session, todayDate)
+  // displayPastTrips()
   // session.getUpcomingTrips(todayDate)
   // toggleView()
   // renderDom(data)
 }
 
+
+
 const changeFormView = () => {
   switch (tripView.value) {
   case 'past':
```

Comment:
great use of switch case!

Reply:
thank you - making the changes on my local repo!